requirements.txt

```diff
@@ -1,6 +1,5 @@
-numpy>=1.12.1
-pandas>=0.23.0
-scikit-learn>=0.18.1
-scipy>=0.19.0
-statsmodels>=0.8.0
-numpydoc>=0.6.0
```

Comment:
@solegalli do you need `numpydoc`?

Reply:
No just for the docs, but I have that in a separate req file inside the doc folder. I was going to commit req files last. But you took care of it. Thanks a lot. Cheers
Sole